libs/langgraph/src/pregel/io.ts

```diff
@@ -209,63 +215,83 @@ export function* mapOutputUpdates<N extends PropertyKey, C extends PropertyKey>(
   if (!outputTasks.length) {
     return;
   }
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  let updated: [N, Record<string, any>][];
+
+  let updated: [N, Record<string, unknown>][];
+
   if (
     outputTasks.some(([task]) =>
       task.writes.some(([chan, _]) => chan === RETURN)
     )
   ) {
+    // TODO: probably should assert that RETURN is the only "non-special" channel (starts with "__")
```

Comment:
left this in on purpose - current implementation is paralleled with python, but it'd be good to add more validation logic in `_commit` in general.